src/app/game-manager/game-manager.service.ts

```diff
@@ -0,0 +1,46 @@
+import { TimerService } from './../shared/timer/timer.service';
+import { Injectable } from '@angular/core';
+import { Observable, ReplaySubject } from 'rxjs';
+import { DataRepository } from '../repository/repository.service';
+import { Game } from '../shared/models/game.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class GameManagerService {
+  private currentGame = new ReplaySubject<Game>();
+  timerTimeout!: ReturnType<typeof setTimeout>;
+  gameWillStartSoon = false;
+  gameIsStarted = false;
```

Comment:
keep game status/state as 1 property and store it in observable, because it will change in time, no need to have 2 flags, it will be hard to correlate them

Reply:
done